src/molecules/Modal/modal.module.scss

```diff
@@ -87,8 +87,14 @@ $border: 1px solid color('neutral-5');
 }
 
 @media #{$small-only} {
-  .modal {
-    padding: 1rem;
+  .modal, .section, .header {
+    padding: ru(1);
```

Comment:
is this value following the Zeplins? PT says padding should be smaller on mobile, but this increases it